routes/flag.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const authenticate = require("../middlewares/authenticate");
+const { authorizeUser } = require("../middlewares/authorization");
+const flagController = require("../controllers/flag");
+const { addFeatureFlagValidator } = require("../middlewares/validators/flag");
+
+router.post("/add", authenticate, authorizeUser("superUser"), addFeatureFlagValidator, flagController.addFlag);
```

Comment:
Can we use the global constant here, instead of "superUser"?

Reply:
Are you suggesting to do as follow :
const superUser = "superUser";
authorizeUser(superUser)

Reply:
Yes. Check `constants` > `role.js`